src/test/run-make/tools.mk

```diff
@@ -1,3 +1,6 @@
+# Default to `/tmp` if no other tmp directory is specified.
+TMPDIR ?= "/tmp"
```

Comment:
Were you running tests manually? In theory should should always be set by the harness running these tests.